openjdk8/src/test/workspace/entrypoint-shutdown-logging.bash

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+sed 's/exec /# /' /docker-entrypoint.bash > /tmp/entrypoint.bash
```

Comment:
It would be great to not have to do this... If you moved this logic outside of the `docker-entrypoint.bash`, would you be able to test the source unmodified?